templates/org/aac/aac-solution-idea.md

```diff
@@ -1,114 +1,129 @@
+- Use the *FirstName LastName* format.
+-->
 
 - [Author 1 Name](https://linkedin.com/in/ProfileURL) | Title, such as "Cloud Solution Architect"
 - [Author 2 Name](https://linkedin.com/in/ProfileURL) | Title, such as "Cloud Solution Architect"
-- > Continue for each primary author (even if there are 10 of them).
 
-Other contributors: > (If applicable.) Include contributing (but not primary) authors, major editors (not minor edits), and technical reviewers. Listed alphabetically by last name. Use this format: Fname Lname. It's okay to add in newer contributors.
+Other contributors: 
+
+<!--
+- This section is optional. 
+- List contributors and technical reviewers. 
+-->
 
 - [Contributor 1 Name](https://linkedin.com/in/ProfileURL) | Title, such as "Cloud Solution Architect"
 - [Contributor 2 Name](https://linkedin.com/in/ProfileURL) | Title, such as "Cloud Solution Architect"
-- > Continue for each additional contributor (even if there are 10 of them).
 
 *To see non-public LinkedIn profiles, sign in to LinkedIn.*
 
 ## Next steps
 
-> Links to articles on Microsoft Learn. Could also be to appropriate sources outside of Learn, such as third-party documentation, GitHub repos, or an official technical blog post.
+<!--
+- Add a bulleted list of links to third-party or Microsoft topics that can help customers build the workload.
+- Link formats: 
+  - Make Learn links site relative (for example, /azure/<feature>/<article-name>).
+  - Start third-party links with `https://` and omit `en-us` unless the links don't work without it.
+  - Omit a trailing slash.
 
-Examples:
+Example list:
 
 - [Azure Kubernetes Service (AKS) documentation](/azure/aks)
 - [Azure Machine Learning documentation](/azure/machine-learning)
-- [What are Azure AI services?](/azure/ai-services/what-are-ai-services)
+- [What is Azure Active Directory B2C?](/azure/active-directory-b2c/overview)
+- [Application Insights overview](/azure/azure-monitor/app/app-insights-overview)
+-->
 
 ## Related resources
 
-> Use "Related resources" for architecture information that's relevant to the current article. It must be content that the Azure Architecture Center TOC refers to, but may be from a repo other than the AAC repo.
-> Links to articles in the AAC repo should be repo-relative, for example (../../solution-ideas/articles/article-name.yml).
+<!-- Add a bulleted list of links to related architecture information in the AAC TOC.
 
-> Here are example sections:
+Example lists:
 
 Related architecture guides:
 
-- [Artificial intelligence (AI) - Architectural overview](/azure/architecture/data-guide/big-data/ai-overview)
-- [Choosing a Microsoft AI services technology](/azure/architecture/data-guide/technology-choices/cognitive-services)
+- [Artificial intelligence (AI) architecture design](/azure/architecture/ai-ml)
+- [Choose a Microsoft AI services technology](/azure/architecture/data-guide/technology-choices/ai-services)
 
 Fully deployable architectures:
```

Comment:
I don't ever see articles with these separate lists, "Fully deployable architectures" and "Related architecture guides." I'd just make one list without an intro phrase for simplicity.